build.gradle

```diff
@@ -20,7 +20,8 @@ ext {
 
   // Third parties
   offheapVersion = '2.2.2'
-  statisticVersion = '1.1.0'
+  managementVersion = '5.0.0.beta'
+  statisticVersion = '1.1-SNAPSHOT'
```

Comment:
This seems to indicate a new version of stats is required. With a SNAPSHOT here PR will never build.
What are the required statistics changes? Is there an issue and a PR opened already?